src/react-apps/applications/service-development/src/features/deploy/components/deployPaper.tsx

```diff
@@ -0,0 +1,403 @@
+/* tslint:disable:max-line-length */
```

Comment:
Can this component be splitt into multiple components? one for each status or something?

Reply:
A very good question.

There is some parts that is shared between all statuses, and some parts that are variants in the different statuses. The statuses are not mutually exclusive in content. So it's a bit tricky to share content if there is several components without duplicating code, and potentially updating several components on update.

Reply:
Personal preference for me is to split this into mulitple components, makes it easier to read and find the code you want to edit

Reply:
Agree. But it is not really multiple components, it's one compoent with variations. It would be a lot of duplication and har do manage changes.

Reply:
Unless by "status" you mean the different "checks" (sync, compile, etc)? That can be arranged. I will put it in outstanding task list. The whole page will be rearranged when we're going to support multiple environments.